webviz_ert/controllers/multi_response_controller.py

```diff
@@ -196,4 +202,32 @@ def _generate_plot(ensemble_id: str, color: str) -> Optional[ResponsePlotModel]:
             for observation in plot._observations:
                 fig.add_trace(observation.repr)
         fig.update_layout(assets.ERTSTYLE["figure"]["layout"])
+        fig.update_layout({"xaxis": {"title": {"text": x_axis_label}}})
+        fig.update_layout(assets.ERTSTYLE["figure"]["layout-value-y-axis-label"])
         return fig
+
+
+def axis_label_for_ensemble_response(
+    ensemble: "EnsembleModel", responseName: str
```

Comment:
should use snake_case: responseName -> response_name

Reply:
fixed, thank you!